modules/api1/docs.md

```diff
@@ -57,6 +57,32 @@ You may pass additional parameters when querying a list:
   returned.
 - `yiiVersion` - version of the framework project built with. Either `1.0` or `1.1`.
 
+
+### Get primary image <a href="#projects-view-primary-image" id="projects-view-primary-image">#</a>
```

Comment:
I agree. I saw that the project is already return the link to the primary image as "thumbnail". I removed an additional query to get the primary image.